.github/scripts/download_pretrained.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import time
+from typing import Optional, Text, Tuple
+
+from transformers import AutoTokenizer, TFAutoModel
+
+import rasa.shared.utils.io
+from rasa.nlu.utils.hugging_face.registry import model_weights_defaults
+
+COMP_NAME = "LanguageModelFeaturizer"
+DEFAULT_MODEL_NAME = "bert"
+
+
+def get_model_name_and_weights_from_config(
+    config_path: str,
+) -> Tuple[Optional[Text], Optional[Text]]:
+    config = rasa.shared.utils.io.read_config_file(config_path)
+    print(config)
+    steps = config.get("pipeline", [])
+
+    # Look for LanguageModelFeaturizer
+    steps = list(filter(lambda x: x["name"] == COMP_NAME, steps))
+
+    if len(steps) == 0:
+        print(f"No {COMP_NAME} found")
+        return None, None
+    elif len(steps) > 1:
```

Comment:
I haven't seen any example of this in an actual config, but in theory it would be allowed to have two `LanguageModelFeaturizer` components in there I think. In this case, all specified models could be downloaded. Not sure what an actual use case would be though.

Reply:
I adjusted the code to a consider a list of `LanguageModelFeaturizer`s to loop over.